### PR TITLE
424 style make footer consistent with rules footer

### PR DESCRIPTION
**Pull request by Geordie88**
cc: @adamcogan, @tiagov8

Hi Tiago,

As per our conversation I have restyled the Peoples footer so that it is consistent with the Rules footer. 

image
**Figure: current People footer**

image
**Figure: Updated peoples footer**

image
**Figure: rules footer**

I could not find an equivalent to https://github.com/SSWConsulting/SSW.Rules/releases/tag/20201117.1 for people so I did not add a link to the build version. 

You will note that in my updated version the build version number has been replaced with "#(VERSION DEPLOYED)" I have not made any changes to cause this so I believe that it is due to my local branch. (I consulted Jack P and he agreed)

### CODE REVIEW

**Review by tiagov8 — CHANGES_REQUESTED** [review]
Hi @Geordie88 

I am happy with the specific changes to the footer... however I am seeing the "bad" changes for the "view-all" link in this PR. More details: https://youtu.be/mdoxkF0s0mo

1. Please chase up @bradystroud and ask him what is the best way to keep your footer changes, but ignore crossed below in this PR:

Screen Shot 2022-08-03 at 2 07 25 PM

**Comment by bradystroud** [comment]
@tiagov8 waiting on your approval

**Review by bradystroud — APPROVED** [review]

**Review by tiagov8 — APPROVED** [review]

**Comment by tiagov8** [comment]
> @tiagov8 waiting on your approval

done

**Pull request merged**
